src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -292,6 +301,96 @@ message MethodConfigProto {
 
 }
 
+// `SampleValueSet` defines a set of parameter values used in
+// generating samples.
+message SampleValueSet {
+  // An identifier, unique to this method, for this set of values.
+  string id = 1;
+
+  // A short, user-visible name for this set of values. This name may
+  // be used in a UI menu, for example.
+  string title = 2;
+
+  // A longer description for this set of values. This name may be
+  // used in a UI hoverbox, for example.
+  string description = 3;
+
+  // A set of parameter values to be used together in samples. Each
+  // member of `parameters` can be specified in one of two formats:
+  //
+  // * `DOTTED.RESOURCE.NAME=VALUE`, to specify the entire value of
+  //   `NAME`.  If `VALUE` is not specified, it is treated as the
+  //    default zero value for `NAME`'s type.
+  // *  `DOTTED.RESOURCE.NAME%FIELD=VALUE`, if `NAME` is required to be
+  //    a specially formatted string using `FIELD` as one of possibly
+  //    several placeholders.
```

Comment:
Your indentation is a little funky above - it transitions from 1 space in from the stars to 2 spaces in from the stars.

Reply:
Fixed. Thanks for the catch!